### PR TITLE
Update submodules

### PR DESCRIPTION
**Pull request by charleslavon**

### CODE REVIEW

**Comment by jackson-harris-iii** [comment]
Are the changes primarily in the submodules repo or are those already committed and we're just updating the references here to match that repo?

**Review by jackson-harris-iii — APPROVED** [review]
Are the changes primarily in the submodules repo or are those already committed and we're just updating the references here to match that repo?

File: .github/workflows/update_submodules.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
   update-submodules:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```
Comment (shelegdmitriy) [inline .github/workflows/update_submodules.yml:12]: Does this change fix [this](https://github.com/near/near-discovery/actions/workflows/update_submodules.yml) failing workflow?

**Comment by charleslavon** [comment]
> Are the changes primarily in the submodules repo or are those already committed and we're just updating the references here to match that repo?

The latter!

File: .github/workflows/update_submodules.yml (hunk 1 above)
Comment (charleslavon) [reply .github/workflows/update_submodules.yml:12]: this along with the addition of the expected REPO_TOKEN should fix it.

**Pull request merged**